list/promise/promise.js

```diff
@@ -1,4 +1,4 @@
-steal("can/list", function (list) {
```

Comment:
In this case, 'list' was referenced but never used. 'can' was not referenced but used.